### PR TITLE
Update 'Admission Control' page for Cloud 2.0

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-9866

### CODE REVIEW

**Comment by rmloveland** [comment]
@florence-crl this is my first ever cloud 2.0 PR! please let me know if I'm holding it wrong :-)

File: src/current/v23.1/admission-control.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Admission control is not available for CockroachDB {{ site.data.products.serverl
 
 A well-provisioned CockroachDB cluster may still encounter performance bottlenecks at the node level, as stateful nodes can develop [hot spots]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#hot-spots) that last until the cluster rebalances itself. When hot spots occur, they should not cause failures or degraded performance for important work.
 
-This is particularly important for CockroachDB {{ site.data.products.serverless }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
+This is particularly important for CockroachDB {{ site.data.products.standard }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
```
Comment (florence-crl) [inline src/current/v23.1/admission-control.md:18]:
```suggestion
This is particularly important for CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
```

File: src/current/v23.2/admission-control.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ For more details about how the admission control system works, see:
 
 A well-provisioned CockroachDB cluster may still encounter performance bottlenecks at the node level, as stateful nodes can develop [hot spots]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#hot-spots) that last until the cluster rebalances itself. When hot spots occur, they should not cause failures or degraded performance for important work.
 
-This is particularly important for CockroachDB {{ site.data.products.serverless }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
+This is particularly important for CockroachDB {{ site.data.products.standard }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
```
Comment (florence-crl) [inline src/current/v23.2/admission-control.md:32]:
```suggestion
This is particularly important for CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
```

File: src/current/v24.1/admission-control.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ For more details about how the admission control system works, see:
 
 A well-provisioned CockroachDB cluster may still encounter performance bottlenecks at the node level, as stateful nodes can develop [hot spots]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#hot-spots) that last until the cluster rebalances itself. When hot spots occur, they should not cause failures or degraded performance for important work.
 
-This is particularly important for CockroachDB {{ site.data.products.serverless }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
+This is particularly important for CockroachDB {{ site.data.products.standard }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
```
Comment (florence-crl) [inline src/current/v24.1/admission-control.md:32]:
```suggestion
This is particularly important for CockroachDB {{ site.data.products.standard }} and CockroachDB {{ site.data.products.basic }}, where one user tenant cluster experiencing high load should not degrade the performance or availability of a different, isolated tenant cluster running on the same host.
```

**Review by florence-crl — APPROVED** [review]
LGTM pending one suggestion.

File: src/current/v23.1/admission-control.md (hunk 1 above)
Comment (rmloveland) [reply src/current/v23.1/admission-control.md]: Made this change, thanks!

File: src/current/v23.2/admission-control.md (hunk 2 above)
Comment (rmloveland) [reply src/current/v23.2/admission-control.md]: ✅

File: src/current/v24.1/admission-control.md (hunk 3 above)
Comment (rmloveland) [reply src/current/v24.1/admission-control.md]: ✅

**Pull request merged**